src/Core/Domain/Feature/Command/EditFeatureCommand.php

```diff
@@ -28,47 +28,48 @@
 
 use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureConstraintException;
 use PrestaShop\PrestaShop\Core\Domain\Feature\ValueObject\FeatureId;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopId;
 
 /**
  * Edit feature with given data.
  */
 class EditFeatureCommand
 {
     /**
-     * @var FeatureId
+     * @var FeatureId|null
```

Comment:
```suggestion
     * @var FeatureId
```
The id is mandatory it cannot be null